LocalSettings.php

```diff
@@ -7,23 +7,13 @@
 /**
  * Configure PHP request timeouts.
  */
- if ( PHP_SAPI === 'cli' ) {
```

Comment:
You remove unsetting the timeout for cli.

Reply:
It said it was probably redundant. Nonetheless, I'll readd it.